components/tensorflow-notebook-image/Dockerfile

```diff
@@ -62,16 +65,6 @@ RUN export CLOUD_SDK_REPO="cloud-sdk-$(lsb_release -c -s)" && \
     gcloud config set component_manager/disable_update_check true && \
     gcloud config set metrics/environment github_docker_image
 
-RUN chown -R ${NB_USER}:users /home/${NB_USER}
-
-# Run everything below this as $NB_USER
-USER $NB_USER
-
-WORKDIR $HOME
-
-# Setup work directory for backward-compatibility
-RUN mkdir /home/$NB_USER/work
-
 # Install Tini - used as entrypoint for container
```

Comment:
Might be worth adding a comment that we want a single layer to minimize size so that people don't split it up later into multiple commands not realizing the benefit of using a single layer.

Reply:
Done.